### ISSUE TITLE
Support for Mintty (Cygwin)

### ISSUE THREAD

**Issue by DiogoDoreto** [larsenwork/monoid#99]
Monoid doesn't shows up on font selection on Mintty. Searching the web I found that Windows have some specific criterias for terminal fonts [1]. Is it practicable to add support?

[1] http://superuser.com/a/689890

**Comment by larsenwork** [comment]
You'd have to open it in a font editor such as fontforge and delete the ligatures (which won't work anyways) as they don't comply to "negative A or C space" rule. Other than that it should work.

**Comment by DiogoDoreto** [comment]
I need to do this even if downloading with ligatures turned off?
Em 30/07/2015 11:36, "Andreas Larsen" <notifications@github.com> escreveu:

**Comment by larsenwork** [comment]
Yep, ligatures off doesn't delete the actual glyphs - only "turns off" the feature.

**Comment by NetVicious** [comment]
Hi! Any tip about how to delete the ligatures using a Font Editor? I downloaded Font Forge Portable and I see lots of glymphs but any tip of which are the ligatures.

**Comment by larsenwork** [comment]
I'm using this: http://fontforge.github.io/en-US/

Ligatures are at the very end together with alternates. Just select them and cmd+x (ctrl+x). 
![screen shot 2015-07-30 at 21 11 29](https://cloud.githubusercontent.com/assets/3859425/8992581/a331ae32-36ff-11e5-85f7-4c7a3685d18a.png)

**Comment by NetVicious** [comment]
Yes, I saw they and cleared all of them. But which options we need to set on the Generate Font window to get a monospaced font ?

**Comment by larsenwork** [comment]
It was already monospaced - it just didn't live up to the weird "negative A or C space" rule. 

I forgot about the W - by default it has a bit overlap to the right. Same for Wcircumflex.

**Comment by larsenwork** [comment]
closed due to inactivity and not a priority for me :)

**Comment by ninrod** [comment]
That's a shame. I love your font and I'm forced to use cygwin at work.

**Comment by ninrod** [comment]
The retina `test` font is working on cygwin though.

**Comment by drasko** [comment]
@larsenwork I deleted all ligatures as you proposed and re-installed Monoid-Regular.ttf - font is still not visible in `mintty`.

Any further instructions?

**Comment by larsenwork** [comment]
@drasko I'll have a cleanup look at the font over the next couple of weeks so I've reopened this and I'll ping you if/when I have something for you to try out 😄
